compiler/src/test/java/com/squareup/anvil/compiler/internal/reference/TopLevelPropertyReferenceTest.kt

```diff
@@ -0,0 +1,189 @@
+                          .isEqualTo("kotlin.Unit")
+                    }
+                  }
+
+                  "prop2" -> {
+                    assertThat(ref.fqName).isEqualTo(FqName("com.squareup.test.prop2"))
+                    assertThat(ref.annotations).isEmpty()
+                    assertThat(ref.getterAnnotations).isEmpty()
+                    assertThat(ref.setterAnnotations).isEmpty()
+                    assertThat(ref.isLateinit()).isFalse()
+                    assertThat(ref.visibility()).isEqualTo(PUBLIC)
+                    assertThat(ref.type().asClassReference().fqName.asString())
+                      .isEqualTo("kotlin.String")
+                  }
+
+                  "prop3" -> {
+                    assertThat(ref.fqName).isEqualTo(FqName("com.squareup.test.prop3"))
+                    assertThat(ref.annotations.single().fqName.asString())
+                      .isEqualTo("kotlin.PublishedApi")
+                    assertThat(ref.getterAnnotations).isEmpty()
+                    assertThat(ref.setterAnnotations).isEmpty()
+                    assertThat(ref.isLateinit()).isFalse()
+                    assertThat(ref.visibility()).isEqualTo(INTERNAL)
+                    assertThat(ref.type().asClassReference().fqName.asString())
+                      .isEqualTo("kotlin.Int")
+                    assertThat(ref.type().isNullable()).isTrue()
+                  }
+
+                  "prop4" -> {
+                    assertThat(ref.fqName).isEqualTo(FqName("com.squareup.test.prop4"))
+                    assertThat(ref.annotations.single().fqName.asString())
+                      .isEqualTo("javax.inject.Inject")
+                    assertThat(ref.getterAnnotations).isEmpty()
+                    assertThat(ref.setterAnnotations.single().fqName.asString())
+                      .isEqualTo("javax.inject.Inject")
+                    assertThat(ref.isLateinit()).isFalse()
+                    assertThat(ref.visibility()).isEqualTo(PUBLIC)
+                    assertThat(ref.type().asClassReference().fqName.asString())
+                      .isEqualTo("kotlin.String")
+                  }
+
+                  "prop5" -> {
+                    assertThat(ref.fqName).isEqualTo(FqName("com.squareup.test.prop5"))
+                    assertThat(ref.annotations.single().fqName.asString())
+                      .isEqualTo("javax.inject.Inject")
+                    assertThat(ref.getterAnnotations.single().fqName.asString())
+                      .isEqualTo("javax.inject.Inject")
+                    assertThat(ref.setterAnnotations).isEmpty()
+                    assertThat(ref.isLateinit()).isFalse()
+                    assertThat(ref.visibility()).isEqualTo(PUBLIC)
+                    assertThat(ref.type().asClassReference().fqName.asString())
+                      .isEqualTo("kotlin.String")
+                  }
+
+                  "prop6" -> {
+                    // lateinit var prop6: String
```

Comment:
Any reason to leave this comment here compared to the other test cases?

Reply:
Forgot to remove, only added it because my screen was too small.